scripts/openduck.py

```diff
@@ -0,0 +1,743 @@
+            modes.choices['amber-prepare'].error('The parameters --ligand, --interaction and --receptor are required.')
+        else:
+            # all good
+            pass
+    elif args.mode == 'Report':
+        if args.output == 'stdout': args.output = sys.stdout # little trick to print
+        if (args.iterations != 20 or args.subsample_size != 20) and args.data not in ('all', 'jarzynski'):
+            modes.choices['report'].error('Iterations and subsample size affect bootstrapping which is only performed when doing jarzynski analysis.')
+        if  args.format == 'openmm' and args.data in ('all', 'jarzynski') and args.step_threshold > 1250:
+            print('OpenMM duck output has 1250 report steps. The index_threshold to find the minima needs to be a >1250. It will be changed to 600.')
+            args.index_threshold == 600
+    elif args.mode == 'Chunk':
+        if (args.yaml_input is None) and (args.ligand is None or args.interaction is None or args.receptor is None):
+            modes.choices['chunk'].error('The input needs to be either the input yaml or specified in the command line (ligand, receptor interaction).')
+        elif args.yaml_input:
+            input_arguments = yaml.load(open(args.yaml_input), Loader=yaml.FullLoader)
+            if all(item in list(input_arguments.keys()) for item in ['receptor_pdb', 'interaction', 'ligand_mol']):
+                #transfer all required items
+                args.ligand = str(input_arguments['ligand_mol'])
+                args.receptor = str(input_arguments['receptor_pdb'])
+                args.interaction = str(input_arguments['interaction'])
+                
+                # overwrite the defaults from command line args
+                if 'cutoff' in input_arguments: args.cutoff =  float(input_arguments['cutoff'])
+                if 'ignore_buffers' in input_arguments: args.ignore_buffers =  bool(input_arguments['ignore_buffers'])
+                if 'output' in input_arguments: args.output =  str(input_arguments['output'])
+
+
+            else:
+                modes.choices['chunk'].error('You need to specify at least "ligand_mol", "receptor_pdb" and "interaction" in the yaml file.')
+        elif (args.ligand is None or args.interaction is None or args.receptor is None):
+            modes.choices['chunk'].error('The parameters --ligand, --interaction and --receptor are required.')
+        else:
+            # all good
+            pass
+        pass
+    return args
+
+def parse_input():
+    ''' Main openduck parser, subparsers define action modes
+    '''
+    parser = argparse.ArgumentParser(description='Open Dynamic Undocking')
+    parser.set_defaults(mode=None)
+    modes = parser.add_subparsers(title='Open Dynamic Undocking toolkit. Choose one of the following actions', help='', metavar='')
+    
+    #Arguments for OPENMM_PREPARATION
+    openmm_prep = modes.add_parser('openmm-prepare', help='Preparation of systems for OpenMM simulations.')
```

Comment:
```suggestion
    openmm_prep = modes.add_parser('openmm-prepare',
                                   help='Preparation of systems for OpenMM simulations.',
                                   description='Preparation of systems for OpenMM simulations' )
```
@AlvaroSmorras could you add a description field to the subparsers as well like so? Then it will appear in the subcommand help as well:
```
usage: openduck openmm-prepare [-h] [-y YAML_INPUT] [-l LIGAND] [-i INTERACTION] [-r RECEPTOR] [--do-chunk] [-c CUTOFF] [-b] [-f {SMIRNOFF,GAFF2}] [-w {tip3p,spce}] [-ff {amber99sb,amber14-all}]
                               [-ion IONIC_STRENGTH] [-s SOLVENT_BUFFER_DISTANCE] [-water WATERS_TO_RETAIN] [-fl] [--do-equilibrate] [-F FORCE_CONSTANT_EQ] [-g GPU_ID]

Preparation of systems for OpenMM simulations.

optional arguments:
  -h, --help            show this help message and exit

...
```
I think this is very useful for users to get a quick idea of what the subcommand does. It could be nice if the `description` was even a bit longer (2-3 sentences) than the `help` field, but duplicating it is fine too.